python/tank/bootstrap/manager.py

```diff
@@ -734,3 +738,31 @@ def _default_failed_callback(self, phase, exception):
         phase_name = "TOOLKIT_BOOTSTRAP_PHASE" if phase == self.TOOLKIT_BOOTSTRAP_PHASE else "ENGINE_STARTUP_PHASE"
 
         log.debug("Default failed callback (%s): %s" % (phase_name, exception))
+
+    def _log_bootstrap_metric(self, engine_name, entity):
+        """
+        Logs a user activity metric when an engine is bootstrapped.
+
+        For example:
+            Module: ``tk-bootstrap basic.maya`` or ``tk-bootstrap classic``
+            Action: ``Launched engine tk-maya in project context``
+
+        :param engine_name: Name of the engine being bootstrapped.
+        :param entity: Shotgun entity to bootstrap into, or ``None`` for the site.
+        """
+
+        # Perform an absolute import to ensure we get the new swapped core.
+        # This is required to make sure we log into the logging queue of this swapped core.
+        from tank.util import log_user_activity_metric
+
+        if self._plugin_id:
+            module = "tk-bootstrap %s" % self._plugin_id
+        else:
+            module = "tk-bootstrap classic"
+
+        context_name = "project" if entity else "site"
+        action = "Launched engine %s in %s context" % (engine_name, context_name)
```

Comment:
Totango is limited in terms of the number of characters you can user per log. For consistency, I'd suggest making the module be `tk-core` since that's where this originates. And I'd make the action be `bootstrap <bootstrap_type> <engine_name> <context_name>`. 

Might be good to followup with @robblau to double check the results in Totango to make sure it works for him in terms of filtering and viewing the data.